es_client.py

```diff
@@ -1,32 +1,47 @@
 # from datetime import datetime
-from elasticsearch_dsl import Document, Integer, Date, Text, Search, Float
+from elasticsearch_dsl import Document, InnerDoc, Integer, Date, Text, Search, Float
 
 
 class ESTweet(Document):
-    id_str = Integer()
     created_at = Date()
     stored_at = Date()
     full_text = Text(analyzer='snowball')
     subjectivity = Float()
     polarity = Float()
+    author_id = Integer()
+    author_followers = Integer()
+
+    class Index:
```

Comment:
add comments on what Index is and its attributes

Reply:
Sure thing! Comments will be added in the next commit.